datastore/api/QuickStart/QuickStart.cs

```diff
@@ -22,11 +23,11 @@ public class QuickStart
     {
         public static void Main(string[] args)
         {
-            // Your Google Cloud Platform project ID
-            string projectId = "YOUR-PROJECT-ID";
-
+            var credential = GoogleCredential.FromEnvironment();
             // Instantiates a client
-            DatastoreDb db = DatastoreDb.Create(projectId);
+            DatastoreDb db = DatastoreDb.Create(credential);

```

Comment:
Isn't projectId embedded in JWT that is used when you generate credential using *.FromEnvironment method?
If yes, we shouldn't have an overload that allows specifying project id - not only it's a duplicate information, but customer may specify id that is incompatible with provided credential.